Benchmarks/Schema.NET.Benchmarks/Schema.NET.Benchmarks.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="Build">
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
```

Comment:
To get this to build on linux, we will need to use mono. To get that working you just need to add this NuGet package:

https://github.com/Dotnet-Boxed/Framework/blob/8acfe10db7d51e1854462a66b11f5a0c8d41965f/Benchmarks/Boxed.Mapping.Benchmark/Boxed.Mapping.Benchmark.csproj#L20

Reply:
Ahhh - that makes sense. I knew the benchmark wouldn't run on Linux when targeting .NET Framework however completely escaped my mind that it wouldn't build.